### PR TITLE
Changed CL to PR and adjusted related terminology

### PR DESCRIPTION
**Pull request by zibellino**
TODO: Change `pr-descriptions.md` to match current practices (eg. reference to **Jira** ticket).

### CODE REVIEW

File: review/developer/pr-descriptions.md
Comment (oestoer) [inline review/developer/pr-descriptions.md:16]: Should this be the title of the PR in our case instead of first line?

File: README.md
Comment (oestoer) [inline README.md]:
```suggestion
    Other organizations often call this a "change", "patch", or "changelist".
```

File: review/developer/pr-descriptions.md
Comment (oestoer) [inline review/developer/pr-descriptions.md:140]: Should we add a section before this that clarifies what are the merging strategies we're supporting?

File: review/developer/small-prs.md
Comment (oestoer) [inline review/developer/small-prs.md:59]: For me this is an important section and addresses a few things that came up in the discussion.
`1 PR != 1 feature`
So this makes the debate about squash or not squash less relevant, and a matter of personal taste (given you don't rewrite the history of a shared branch like master)

File: review/reviewer/speed.md
Comment (oestoer) [inline review/reviewer/speed.md]: This sentence got shortened a bit and for me it's not clear now with the modification. Was this intentional or accidental?

**Review by oestoer — APPROVED** [review]
Thanks Szabolcs for updating this. Looks good and I've added a few comments and questions.

File: review/developer/pr-descriptions.md
Comment (zibellino) [reply review/developer/pr-descriptions.md:140]: @oestoer We definitely should, do you have something in mind?

File: review/reviewer/speed.md
Comment (zibellino) [reply review/reviewer/speed.md]: fixed

File: review/developer/pr-descriptions.md
Comment (zibellino) [reply review/developer/pr-descriptions.md:16]: It probably should, and it should probably also mention the Jira ticket references at some point too.
Shall I sketch up something?

**Review by jury89 — APPROVED** [review]
LGTM

File: review/developer/pr-descriptions.md
Comment (oestoer) [reply review/developer/pr-descriptions.md:16]: yes, it's okay if it's just a draft, also it doesn't block this PR

File: review/developer/pr-descriptions.md
Comment (oestoer) [reply review/developer/pr-descriptions.md:140]: yes, I'll create a separate PR for that

**Comment by zibellino** [comment]
TODO will be added as ticket to the BE Jira project (to be created).

**Pull request merged**
